### PR TITLE
SimpleCalc Sample App - Single Project Migration

### PR DESCRIPTION
**Pull request by LProulx86**
migration sample app : https://github.com/unoplatform/uno.samples-private/issues/146

remove old sample simple calculator app.

Add new 4 versions of single app sample : simple calculator.

### CODE REVIEW

File: reference/SimpleCalc/MVVM-Xaml/SimpleCalculator.Tests/SimpleCalculator.Tests.csproj
Comment (eriklimakc) [inline reference/SimpleCalc/MVVM-Xaml/SimpleCalculator.Tests/SimpleCalculator.Tests.csproj:1]: I think we are missing the tests projects for all 4 versions of SimpleCalc.

**Comment by jeromelaban** [comment]
@LProulx86 make sure to rename the sln files so they get build by the CI.

File: reference/SimpleCalc/README.md [hunk 1]
```diff
@@ -1,10 +1,7 @@
...
 
-  <!-- Disabling source code updating - uncomment and do build to update source code from shared resources -->
-  <!-- <Import Project="..\..\resources\shared-resources.targets" Condition="Exists('..\..\resources\shared-resources.targets')" /> -->
+This app was developed using Uno Platform and draws inspiration from [@adolfomarinucci's](https://x.com/adolfomarinucci) [Calculator project.](https://github.com/adospace/reactorui-maui/tree/main/samples/Calculator)
 
-Make changes to the shared source code in the `resources` folder and then build the solution. The shared source code will be copied into the solution. Commit the changes to the solution and the shared source code.
-
-Don't forget to update the other solutions that will be affected eg if you make changes to the XAML, make sure both the XAML solutions are updated.
\ No newline at end of file
+![SimpleCalculator light Image](simplecalculator-light.png) ![SimpleCalculator dark Image](simplecalculator-dark.png)
```
Comment (eriklimakc) [inline reference/SimpleCalc/README.md:7]: Can we also add the workshop link (https://platform.uno/docs/articles/external/workshops/simple-calc/README.html) as we did with TubePlayer [here](https://github.com/unoplatform/Uno.Samples/blob/feature/singleproject/reference/TubePlayer/README.md?plain=1#L43-L45) ?

File: reference/SimpleCalc/CSharp-MVUX/SimpleCalculator.sln
Comment (eriklimakc) [inline reference/SimpleCalc/CSharp-MVUX/SimpleCalculator.sln:1]: Remember to rename the sln files to match what Jerome did on this commit (https://github.com/unoplatform/Uno.Samples/pull/669/commits/679a4c320276088f5b04b29f44ce9f7823b3551d) regarding Counter app.

**Review by eriklimakc — APPROVED** [review]

File: reference/SimpleCalc/XAML-MVUX/Directory.Build.props [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+      PRI257: Ignore default language (en) not being one of the included resources (eg en-us, en-uk)
+    -->
+    <NoWarn>$(NoWarn);NU1507;NETSDK1201;PRI257</NoWarn>
+  </PropertyGroup>
+
+  <PropertyGroup>
+    <UnoExtensionsVersion>4.1.14</UnoExtensionsVersion>
+    <UnoToolkitVersion>6.0.18</UnoToolkitVersion>
+    <UnoThemesVersion>5.0.13</UnoThemesVersion>
+    <UnoCSharpMarkupVersion>5.2.13</UnoCSharpMarkupVersion>
```
Comment (vatsashah45) [inline reference/SimpleCalc/XAML-MVUX/Directory.Build.props:20]: Do we need C#Markup for the XAML-MVUX Project?

File: reference/SimpleCalc/XAML-MVVM/Directory.Build.props [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+      PRI257: Ignore default language (en) not being one of the included resources (eg en-us, en-uk)
+    -->
+    <NoWarn>$(NoWarn);NU1507;NETSDK1201;PRI257</NoWarn>
+  </PropertyGroup>
+
+  <PropertyGroup>
+    <UnoExtensionsVersion>4.1.14</UnoExtensionsVersion>
+    <UnoToolkitVersion>6.0.18</UnoToolkitVersion>
+    <UnoThemesVersion>5.0.13</UnoThemesVersion>
+    <UnoCSharpMarkupVersion>5.2.13</UnoCSharpMarkupVersion>
```
Comment (vatsashah45) [inline reference/SimpleCalc/XAML-MVVM/Directory.Build.props:20]: Do we need C#Markup for the XAML-MVVM project?

File: reference/SimpleCalc/simplecalculator-light.png
Comment (agneszitte) [inline reference/SimpleCalc/simplecalculator-light.png:1]: @LProulx86 can we use better screenshots here please, we have those ones in the workshop
- https://platform.uno/docs/articles/external/workshops/simple-calc/art/Light/iphone14.png
- https://platform.uno/docs/articles/external/workshops/simple-calc/art/Dark/iphone14.png

Also if screenshots are added for the ReadMe, they should be in a proper assets folder please

File: reference/SimpleCalc/XAML-MVUX/Directory.Build.props (hunk 2 above)
Comment (agneszitte) [reply reference/SimpleCalc/XAML-MVUX/Directory.Build.props:20]: cc @eriklimakc

**Pull request merged**